### PR TITLE
fix(icons,forms): styled RadioBtns

### PR DESCRIPTION
**Pull request by eckdanny-osi**
📘 [`CL-213`](https://osi-cwds.atlassian.net/browse/CL-213)

### CODE REVIEW

File: libs/forms/src/RadioGroup.module.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  &:focus-within {
+    background-color: lighten(map-get($map: $theme-colors, $key: 'breath'), 2%);
+  }
+  input[type='radio'] {
+    opacity: 0;
+    text-indent: -99999rem;
+    position: absolute;
+  }
+  input[type='radio']:focus + .RadioIcon {
+    // box-shadow: 0 0 0 $btn-focus-width theme-color($key: accent);
```
Comment (mlipardo) [inline libs/forms/src/RadioGroup.module.scss]: Should be removed then since it's just comments?

File: libs/forms/src/RadioGroup.tsx [hunk 2]
```diff
@@ -23,12 +24,16 @@ export interface RadioGroupProps<T> extends IListType<T> {
 const RadioGroup = (props: RadioGroupProps<any>) => {
   const grpId = `${props.name}-rdo${uniqueId()}`
   return (
-    <FormGroup tag="fieldset">
+    <fieldset>
```
Comment (mlipardo) [inline libs/forms/src/RadioGroup.tsx]: should we use native `fieldset` or our component `fieldset` @cwds/forms?

Comment (eckdanny-osi) [reply libs/forms/src/RadioGroup.tsx]: ⚡️ `@cwds/forms::Fieldset`. tnx!

File: libs/forms/src/RadioGroup.module.scss (hunk 1 above)
Comment (eckdanny-osi) [reply libs/forms/src/RadioGroup.module.scss]: yes

**Pull request merged**